### ISSUE TITLE
Brave shields always down be default on sites, but enabled in global settings

### ISSUE THREAD

**Issue by FearNaBoinne** [brave/brave-browser#24514]
## Description 
Despite being enabled in global settings, any new site loads with shields down, and I have to manually enable them!

## Steps to Reproduce
   1. Enable shields in global settings
   2. Load new site
   3. See that the icon is grey and shields are down

## Actual result:
Shields down

## Expected result:
Shields up

## Reproduces how often: 
100%

## Brave version (info found on brave://version)
Brave | 1.42.86 Chromium: 104.0.5112.81 (Official Build) (64-bit)
Revision | 5b7b76419d50f583022568b6764b630f6ddc9208-refs/branch-heads/5112@{#1309}
OS | Windows 10 Version 21H2 (Build 19044.1826)
JavaScript | V8 10.4.132.20
User Agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/104.0.5112.81 Safari/537.36

## Version/Channel Information:
- Can you reproduce this issue with the current release?
I am on current
- Can you reproduce this issue with the beta channel?
I haven't tried, but it's been several versions now
- Can you reproduce this issue with the dev channel?
I haven't tried, but it's been several versions now
- Can you reproduce this issue with the nightly channel?
I haven't tried, but it's been several versions now

**Comment by FearNaBoinne** [comment]
![2022-08-08 10_29_21-Settings - Brave](https://user-images.githubusercontent.com/22585449/183377836-2bf34cbe-cde1-4c9f-bdae-bf1a9855ae70.png)

**Comment by WalletUpLLC** [comment]
Experiencing the same exact issues for several weeks now. 
[Version 1.42.86 Chromium: 104.0.5112.81 (Official Build) (x86_64](https://brave.com/latest/) on Macbook Pro 11.6.8 (20G730)

**Comment by rebron** [comment]
@FearNaBoinne and @WalletUpLLC Can you post a screenshot of the settings you have in brave://settings/cookies? Don't need to include the customize behaviors section.

**Comment by boocmp** [comment]
@FearNaBoinne and @WalletUpLLC do you use Sync?

**Comment by WalletUpLLC** [comment]
> @FearNaBoinne and @WalletUpLLC Can you post a screenshot of the settings you have in brave://settings/cookies? Don't need to include the customize behaviors section.

<img width="980" alt="Screen Shot 2022-08-09 at 11 58 44 PM" src="https://user-images.githubusercontent.com/81252130/183809959-7336eaf2-c434-47e3-a34a-967b207721ca.png">
Please find in attached. It has been working for several months , till the several past weeks ago. Did not make report because assumed it was a glitch that would be resolved till now.

**Comment by WalletUpLLC** [comment]
> @FearNaBoinne and @WalletUpLLC do you use Sync?

it happens with or without

**Comment by FearNaBoinne** [comment]
> > @FearNaBoinne and @WalletUpLLC Can you post a screenshot of the settings you have in brave://settings/cookies? Don't need to include the customize behaviors section.
> 
> <img alt="Screen Shot 2022-08-09 at 11 58 44 PM" width="980" src="https://user-images.githubusercontent.com/81252130/183809959-7336eaf2-c434-47e3-a34a-967b207721ca.png">
> 
> Please find in attached. It has been working for several months , till the several past weeks ago. Did not make report because assumed it was a glitch that would be resolved till now.

Mine are identical to @WalletUpLLC 



> > @FearNaBoinne and @WalletUpLLC do you use Sync?
> 
> it happens with or without

Same as well, but right now sync is enabled...

**Comment by boocmp** [comment]
@FearNaBoinne Could you please open brave://sync-internals, go to the "Sync Node Browser" tab, expand Preferences and find encrypted node with name:"profile.content_settings.exceptions.braveShields". Does it contain a rule with the key "*,*"?

**Comment by FearNaBoinne** [comment]
> @FearNaBoinne Could you please open brave://sync-internals, go to the "Sync Node Browser" tab, expand Preferences and find encrypted node with name:"profile.content_settings.exceptions.braveShields". Does it contain a rule with the key `"*,*"`? ![image](https://user-images.githubusercontent.com/73575789/183868673-97ebb5f3-ff38-4d0d-b50b-c36d3bf33ad3.png)

There is NO encrypted node with that name, only a normal named node!
(I only have 4 encrypted nodes, "extensions.pinned_extensions", "profile.content_settings.exceptions.popups", "extensions.commands" and "profile.content_settings.exceptions.font_access"

The normal named node does NOT have a "*,*":
![2022-08-10 11_36_56-Sync Internals - Brave](https://user-images.githubusercontent.com/22585449/183870210-d74492da-c15a-41d9-951e-7d30dd6176c6.png)

**Comment by boocmp** [comment]
Thanks a lot! I guess we have found the cause of the problem

**Comment by FearNaBoinne** [comment]
> Thanks a lot! I guess we have found the cause of the problem

And how do we fix it?  :P

**Comment by boocmp** [comment]
Uhhh, I guess disabling synchronization and restarting the browser should fix this. But I'm afraid there is no workaround if you want to use synchronization. I believe we will release a fix in near future.

**Comment by FearNaBoinne** [comment]
Then I will wait for that fix to come around! Thx!

**Comment by WalletUpLLC** [comment]
> Uhhh, I guess disabling synchronization and restarting the browser should fix this. But I'm afraid there is no workaround if you want to use synchronization. I believe we will release a fix in near future.

same in here, but got 38 encrypted, and {\"*,*\" found in an unencrypted profile.content_settings.exceptions.braveShields

However there is a partial workaround. 
Since the issue is narrowed down to the "settings preferences", and assuming that the settings were already sync across the synced devices, you can safely go to brave://settings/braveSync/setup and select the "customize sync" target and disable "settings". Restart brave and you should be good to go. You want to apply same process on the profiles where you want it to load global shield settings 
<img width="1033" alt="Screen Shot 2022-08-10 at 9 08 09 AM" src="https://user-images.githubusercontent.com/81252130/183909625-fe100c34-b360-41ee-ab88-3ce2b24e097e.png">
 
 @FearNaBoinne try and see how it works for you

**Comment by kjozwiak** [comment]
@brave/qa-team added some more detailed/easier to follow STR/Cases via https://github.com/brave/brave-core/pull/14592#issuecomment-1223614343 based on the information from @boocmp via https://github.com/brave/brave-core/pull/14592#issue-1336036929.

**Comment by kjozwiak** [comment]
The above will require `1.43.80` or higher for `1.43.x` verification 👍

**Comment by stephendonner** [comment]
## Verification `IN-PROGRESS` using 

Brave | 1.43.80 Chromium: 104.0.5112.102 (Official Build) dev (x86_64)
-- | --
Revision | 8e5396254975ef939f2ef7d0bd334e48a052b536-refs/branch-heads/5112@{#1478}
OS | macOS Version 11.6.8 (Build 20G730)



Reproduced the issue on `macOS x64` using the following STR/Cases and build(s):

The following STR/Cases were created using the information from https://github.com/brave/brave-core/pull/14592#issue-1336036929 and after speaking with @boocmp.

* downloaded & installed `1.40.113 Chromium: 103.0.5060.114`
  * build without https://github.com/brave/brave-core/pull/14031
* launched & closed `1.40.113 Chromium: 103.0.5060.114`
* opened the `preferences` file via `\`
* edited the `profile.content_settings.exceptions.braveShields` key as per the following:

```
{"*,*":{"expiration":"0","last_modified":"13304670271801570","model":0,"setting":2}}
```

* launched `1.40.113 Chromium: 103.0.5060.114` and ensured that shields were disabled on all websites visited
* enabled sync chain via `brave://settings/braveSync` and enabled `Sync everything` (waited ~2-3mins)
* closed `1.40.113 Chromium: 103.0.5060.114`
* installed `1.44.40 Chromium: 104.0.5112.81` and joined the sync chain that was created with `1.40.113`
* enabled `Sync everything` (waited ~2-3mins)

Once the above was done, any website visited within `1.40.113` also had shields disabled. Examples below re: issue occurring:

#### Issue occurring via `1.40.113 Chromium: 103.0.5060.114`

**`Example`** | **`Example`** | **`Example`**
----------------|-----------------|----------------
release 1 | release 2 | release 3

#### Issue occurring via `1.44.40 Chromium: 104.0.5112.81`

**`Example`** | **`Example`** | **`Example`**
----------------|-----------------|----------------
nightly 1 | nightly 2 | nightly 3

Verification **PASSED** on `Win 11 x64` using the following build(s):

```
.
```

Using the STR/Cases mentioned above, went through the same process but using `1.44.62 Chromium: 104.0.5112.102 ` and ensured that shields were working as expected on `1.44.62 Chromium: 104.0.5112.102` as per the following:

#### Issue occurring via `1.40.113 Chromium: 103.0.5060.114`

**`Example`** | **`Example`** | **`Example`**
----------------|-----------------|----------------
broken 1 | broken 2 | broken 3

#### Shields working as expected via `1.44.62 Chromium: 104.0.5112.102`

**`Example`** | **`Example`** | **`Example`**
----------------|-----------------|---------------
nightly 1 | nightly 2 | nightly 3

Once both versions synced, ensured that shields were also working via `1.40.113 Chromium: 103.0.5060.114`.

**Comment by MadhaviSeelam** [comment]
Verification `PASSED` using
```
Brave | 1.43.81 Chromium: 104.0.5112.102 (Official Build) (64-bit)
-- | --
Revision | 8e5396254975ef939f2ef7d0bd334e48a052b536-refs/branch-heads/5112@{#1478}
OS | Windows 11 Version 21H2 (Build 22000.856)
```

<details>

Reproduced the issue using the following STR/Cases and build(s):

* Installed  `1.40.113 Chromium: 103.0.5060.114`
* launched Brave
*  visited https://cnn.com and disable Shields
* closed `1.40.113 Chromium: 103.0.5060.114`
* edited the `braveShields` entry in `C:\Users\mseel\AppData\Local\BraveSoftware\Brave-Browser\User Data\Default\Preferences`
```
{"*,*": {"expiration":"0","last_modified":"13305939022714591","model":0,"setting":2})
```
* launched `1.40.113 Chromium: 103.0.5060.114` and ensured that shields were disabled on all websites visited
* enabled sync chain via `brave://settings/braveSync` and enabled `Sync everything` (waited ~2-3mins)
* closed `1.40.113 Chromium: 103.0.5060.114`
* installed `1.43.81 Chromium: 104.0.5112.81` and joined the sync chain that was created with `1.40.113` (using the daily code)
* enabled `Sync everything` (waited ~2-3mins)

Once the above was done, any website visited within `1.40.113` also had shields disabled. Examples below re: issue occurring:

Ex1|Ex2|Ex3|Ex4
---|---|---|---
<img width="946" alt="image" src="https://user-images.githubusercontent.com/98358127/186775852-4f87a571-a936-406e-99b4-2ebea257783f.png">|<img width="947" alt="image" src="https://user-images.githubusercontent.com/98358127/186775887-b6101861-18ca-4643-926b-08b434563913.png">|<img width="956" alt="image" src="https://user-images.githubusercontent.com/98358127/186775922-5aa2e989-b08d-4c83-a84e-9d44212975a2.png">|<img width="960" alt="image" src="https://user-images.githubusercontent.com/98358127/186775947-8e958945-8342-4249-9128-52faca95748d.png">

## Verification on a new profile `1.43.81 Chromium: 104.0.5112.102`

- Installed 1.43.81
- launch Brave
- visited https:// cnn.com and disabled shields
- close the browser
- edited the `braveShields` entry in `C:\Users\mseel\AppData\Local\BraveSoftware\Brave-Browser\User Data\Default\Preferences`
   ```
   {"*,*": {"expiration":"0","last_modified":"13305939745950080","model":0,"setting":2})
   ```
- relaunch the Brave
- confirmed Shields is enabled and working as expected. 
- visited other sites and confirmed Shields are enabled and working as expected (nytimes.com) and (bbc.com) 

Ex1|Ex2|Ex3
---|---|---
<img width="950" alt="image" src="https://user-images.githubusercontent.com/98358127/186776177-0b1cfb7f-d2e5-4378-9935-0936338a2a95.png">|<img width="953" alt="image" src="https://user-images.githubusercontent.com/98358127/186776239-c6780442-68fe-405a-814e-5050d4159bf7.png">|<img width="948" alt="image" src="https://user-images.githubusercontent.com/98358127/186776274-e7076e64-6334-4f7c-ad04-e32a323eb1f2.png">
